src/coreclr/jit/gentree.h

```diff
@@ -5259,6 +5259,34 @@ struct GenTreeJitIntrinsic : public GenTreeMultiOp
         return (CorInfoType)gtSimdBaseJitType;
     }
 
+    CorInfoType GetSimdBaseJitPreciseType() const
```

Comment:
Nit: `GetNormalizedSimdBaseJitType` may be more a more clear name.

Reply:
I thought about using the name `Normalized` as well, but we seem to already use `Precise` to describe this, so I went with that. If could honestly go either way, and I almost prefer to use `Normalized` actually, so I can change it to that.

Reply:
> but we seem to already use `Precise` to describe this

Note that the idea behind `Precise` was to preserve the unsigned-ness of "large" types (`UINT`/`ULONG`), as the plain `JITtype2varType` was losing it ("large" unsigned types do not exist in IR). The fact it normalizes `N[U]INT`s is sort of a side-effect (it "has to").